dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -1288,162 +1288,18 @@ def assert_caching_enabled(cache_control_header, max_age, proxy_max_age)
     assert_equal assigns(:level), level2
   end
 
-  def put_student_in_section(student, teacher, script)
-    section = create :section, user_id: teacher.id, script_id: script.id
-    Follower.create!(section_id: section.id, student_user_id: student.id, user: teacher)
-    section
-  end
-
-  def get_hidden(script)
-    response = get :hidden, params: {script_id: script.name}
+  test "hidden_stage_ids for user not signed in" do
+    response = get :hidden_stage_ids, params: {script_id: @script.name}
```

Comment:
I'd add a test for expected behavior if someone tries to call the `hidden_stage_ids` route without the `script_id` param.

Reply:
The param is part of the path, so I'm not sure that's possible, i.e. the route is something like `/s/csp1/hidden_stages`

Reply:
My bad, I misunderstood the route. :+1: